### PR TITLE
Mobile styling fixes

### PR DESCRIPTION
**Pull request by GuillaumeRx**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
image
@GuillaumeRx Should we have the read/write buttons expand to be full width as well?

File: src/features/InteractWithContracts/components/Interact.tsx [hunk 1]
```diff
@@ -390,7 +391,12 @@ function Interact(props: CombinedProps) {
             </FieldWrapper>
 
             <ButtonWrapper>
-              <Button color={WHITE} disabled={wasContractInteracted} onClick={submitInteract}>
+              <Button
+                color={WHITE}
+                disabled={wasContractInteracted}
+                onClick={submitInteract}
+                fullwidth={isMobile}
```
Comment (FrederikBolding) [inline src/features/InteractWithContracts/components/Interact.tsx:398]: Is it only intended to be full width on mobile?

File: src/features/DeployContracts/components/Deploy.tsx [hunk 2]
```diff
@@ -201,7 +203,7 @@ export const Deploy = (props: Props) => {
       )}
 
       <ButtonWrapper>
-        <Button disabled={isDemoMode} onClick={deploySubmit}>
+        <Button disabled={isDemoMode} onClick={deploySubmit} fullwidth={isMobile}>
```
Comment (FrederikBolding) [inline src/features/DeployContracts/components/Deploy.tsx:206]: Same as interact

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/features/InteractWithContracts/components/Interact.tsx (hunk 1 above)
Comment (GuillaumeRx) [reply src/features/InteractWithContracts/components/Interact.tsx:398]: Yes it's only fullwidth on mobile :)

File: src/features/DeployContracts/components/Deploy.tsx (hunk 2 above)
Comment (GuillaumeRx) [reply src/features/DeployContracts/components/Deploy.tsx:206]: yup yup

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**